examples/internal/integration/integration_test.go

```diff
@@ -43,11 +43,15 @@ func TestEcho(t *testing.T) {
 		return
 	}
 
-	testEcho(t, 8088, "application/json")
-	testEchoOneof(t, 8088, "application/json")
-	testEchoOneof1(t, 8088, "application/json")
-	testEchoOneof2(t, 8088, "application/json")
-	testEchoBody(t, 8088)
+	for _, apiPrefix := range []string{"v1", "v2"} {
+		t.Run(apiPrefix, func(t *testing.T) {
+			testEcho(t, 8088, apiPrefix, "application/json")
+			testEchoOneof(t, 8088, apiPrefix, "application/json")
+			testEchoOneof1(t, 8088, apiPrefix, "application/json")
+			testEchoOneof2(t, 8088, apiPrefix, "application/json")
+			testEchoBody(t, 8088, apiPrefix)
+		})
+	}
```

Comment:
Why is the apiPrefix necessary?

Reply:
I want to test api routes described in echo_service.proto and standalone_echo_service.yaml using same tests